pkgs/development/compilers/bigloo/default.nix

```diff
@@ -24,9 +24,10 @@ stdenv.mkDerivation rec {
     # For libuv on darwin
     lib.optionalString stdenv.hostPlatform.isDarwin ''
       export LIBTOOLIZE=libtoolize
-    '' +
+    ''
     # Help libgc's configure.
```

Comment:
While at it, might be better to place it closer to the thing it refers to inside the string, so that it does not get lost.

```suggestion
```